core/src/main/resources/lib/form/secretTextarea/secret.css

```diff
@@ -65,3 +65,7 @@
   border: none;
   padding: 1em;
 }
+
+.secret * [hidden] {
+  display: none !important;
+}
```

Comment:
despite the elements having the `hidden` attributes they where continuing to show up.
I did not need  !important` (they where hidden without it) not did I try and track down exactly what CSS rule was matching so that they did show up.

whilst `!important` is probably bad form - anyone setting a display on somethign with a hidden attribute is probably a bad actor that would need to be fixed.  In any case hidden here means hidden.